includes/ets_user_qa_function.php

```diff
@@ -1,193 +1,194 @@
+<?php
```

Comment:
The blank space on the start of the file was the only change we did here?

Reply:
Yes